docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -783,29 +765,47 @@ kotlin {
     }
 
     // Configure all compilations of all targets:
-    targets.all {
-        compilations.all {
-            compilerOptions.configure {
-                allWarningsAsErrors = true
-            }
-        }
+    compilerOptions {
+        allWarningsAsErrors = true
     }
 }
 ```
 
 </tab>
 </tabs>
 
-Alternatively, to configure compiler options that are common for all targets, you can use the `compilerOptions {}` [top-level block](multiplatform-dsl-reference.md#top-level-blocks):
+## Compiler options
+
+You can configure compiler options in your projects at three different levels:
+
+* **Extension level**, by using it as a top-level block.
+* **Target level**.
```

Comment:
Maybe mention that it is available in the specific target DSL block?

Reply:
Would the below work?
> **Target level**, by using it in the target block.

Reply:
yes